src/components/views/ScreeningsCard.vue

```diff
@@ -0,0 +1,121 @@
+<script>
+import { defineComponent } from 'vue';
+import MoviePoster from '@/components/common/Movie/MoviePoster.vue';
+import MovieLength from '@/components/common/Movie/MovieLength.vue';
+import AppButton from '@/components/common/App/AppButton.vue';
+import MovieTitle from '@/components/common/Movie/MovieTitle.vue';
+import AppTag from '@/components/common/App/AppTag.vue';
+import { getScreeningsByMovieAndDate } from '@/api/service/Screenings';
+
+export default defineComponent({
+  components: { MoviePoster, MovieLength, AppButton, MovieTitle, AppTag },
+  props : {
+    movieData: {
+      type: Object,
+      required: true,
+    },
+    daySelected: {
+      type: Date,
+      default: new Date().toISOString().slice(0, 10)
+    },
+    titleSelected: {
+      type: String,
+      default: 'All movies',
+    }
+  },
+  data() {
+    return {
+      screenings: [],
+      selectedMovieId: this.movieData.id,
+    }
+  },
+  watch: {
+    daySelected(newDay, previousDay) {
+      if (newDay !== previousDay) {
```

Comment:
These checks aren't necessary. The watcher will be invoked only if the value changed.

Reply:
Works indeed without if-ology 😄 👍